### PR TITLE
Pipeline changes and zip with PS.

### PR DESCRIPTION
**Pull request by AlexisErazoGlobant**

### CODE REVIEW

File: pipeline.yml [hunk 1]
```diff
@@ -23,12 +23,34 @@ ci_test:
...
+    fail_on_audit_failure: false
     third_party_lib_paths:
       - package_output
+
+deployment:
+  -  
+    type: sign
+    files_to_sign:
+        - package_output\DSIronPython3\bin\DSIronPythonEmpty.dll
+        - package_output\DSIronPython3\extra\IronPython.dll
```
Comment (mjkkirschner) [inline pipeline.yml]: If this is not our .dll then we should not sign it.

Comment (sm6srw) [reply pipeline.yml]: Correct, we should sign `python3eval.dll`, not `IronPython.dll`

Comment (AlexisErazoGlobant) [reply pipeline.yml]: Solved.

**Review by sm6srw — APPROVED** [review]
LGTM!

**Pull request merged**